doc/sphinx/source/recipes/recipe_clouds.rst

```diff
@@ -317,5 +568,49 @@ Example plots
    Interannual variability of modeled and observed (GPCP) precipitation rates
    estimated as relative temporal standard deviation from 20 years (1986-2005)
    of data. The temporal standard devitions are calculated from monthly
-   anomalies after subtracting the climatological mean seasonal cycle.
+   anomalies after subtracting the climatological mean seasonal cycle. Produced
+   with recipe_lauer13jclim.yml.
+
+.. _fig_cloud_5:
+.. figure::  /recipes/figures/clouds/clouds_zonal_clcalipso_annual_cmip6.png
+   :align:   center
+   :width:   14cm
+
+   Zonal mean of the multi-year annual mean cloud fraction as seen from
+   CALIPSO from CMIP6 models in comparison to CALIPSO-GOCCP data. Produced
+   with recipe_lauer22jclim_fig3-4_zonal.yml.
+
+.. _fig_cloud_6:
+.. figure::  /recipes/figures/clouds/clouds_scatter_clt_swcre_so_cmip6.png
+   :align:   center
+   :width:   10cm
+
+   Multi-year seasonal average (December-January-February) of cloud shortwave
+   radiative effect (W m\ :sup:`-2`) vs. total cloud fraction (clt, %) averaged over the
+   Southern Ocean defined as latitude belt 30°S-65°S (ocean grid cells only).
+   Shown are the CMIP6 multi-model mean (red filled circles and lines) and 
+   observational estimates from ESACCI-CLOUD (black circles and lines).
+   The red shaded areas represent the range between the 10th and 90th percentiles
+   of the results from all individual models. Produced with
+   recipe_lauer22jclim_fig9-11ab_scatter.yml.
+
+.. _fig_cloud_7:
+.. figure::  /recipes/figures/clouds/clouds_pdf_clt_so_cmip6_line.png
+   :align:   center
+   :width:   8cm
+
+   Frequency distribution of monthly mean total cloud cover from CMIP6 models
+   in comparison to ESACCI-CLOUD data. The red curve shows the multi-model average,
+   the blue curve the ESACCI-CLOUD data and the thin gray lines the individual
+   models. The red shading shows ±1 standard deviation of the inter-model spread.
+   Produced with recipe_lauer22jclim_fig9-11c_pdf.yml.
+
+.. _fig_cloud_8:
+.. figure::  /recipes/figures/clouds/clouds_dyn_matrix_ts_wap_clt_cmip6_ocean.png
+   :align:   center
+   :width:   8cm
 
+   2-dimensional distribution of average total cloud cover (clt) binned by sea
+   surface temperature (SST, x-axis) and vertical velocity at 500 hPa (ω\ :sub:`500`, y-axis)
+   averaged over 20 years and all grid cells over the ocean. Produced with
+   recipe_lauer22jclim_fig8_dyn.yml.
```

Comment:
The panels are optional and turned on/off with the recipe settings (parameter "sidepanels"). I updated the example figure now not showing any side panels to avoid confusion.